docs/services.md

```diff
@@ -658,7 +658,7 @@ const credentials = {
   url: '{s3 url}'
 };
 uploadsClient.createUpload({
-  titleset: `${myUsername}.${myTileset}`,
+  tileset: `${myUsername}.${myTileset}`,
```

Comment:
Thanks for the PR, the md files are autogenerated from the JS files, so the corresponding line is at https://github.com/mapbox/mapbox-sdk-js/blob/main/services/uploads.js#L105, could you update it there too?

Reply:
done